app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,20 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
+    end
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
so sánh nil, empty, blank giúp a nè

Reply:
Vang a!

Reply:
- nil? : co the su dung tren bat ki doi tuong nao va chi tra ve true khi no thuc su la nil ( nil.nil? => true )
- empty? : chi co the su dung tren cac doi tuong dang tap hop nhu: array, hash, string.., se tra ve true neu cac tap hop khong co phan tu nao
- blank? : su dung blank? voi nil va false se tra ve true(nil.blank? => true), con true se tra ve false (true.blank?=>false), array va hash la blank khi chung empty ( [ ].blank? => true, { }.blank? true )

Reply:
Vậy ưu tiên sử dụng cái nào ?

Reply:
Nen su dung blank hon.

Reply:
> cập nhâp câu trả lời từng chương lên mô tả pull request nhé

Vang a!